rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -881,6 +881,30 @@ void getActiveFederationCreationBlockHeight_afterRSKIP186AndNoValueInStorage_ret
 
     }
 
+    @Test
+    void testSetActiveFederationCreationBlockHeight() {
```

Comment:
nit: remove the `test` part of the test name

Reply:
Updated.